ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -81,9 +81,6 @@ def _process_trajectory(self, trajectory: Trajectory) -> None:
         super()._process_trajectory(trajectory)
         agent_id = trajectory.agent_id  # All the agents should have the same ID
 
-        # Add to episode_steps
-        self.episode_steps[agent_id] += len(trajectory.steps)
```

Comment:
So this wasn't being used for anything?

Reply:
Left over from when reporting episode steps moved to AgentProcessor